source/extensions/bootstrap/internal_listener/client_connection_factory.h

```diff
@@ -35,6 +37,7 @@ class InternalClientConnectionFactory : public Network::ClientConnectionFactory,
   // TODO(lambdai): make it friend to only bootstrap extension.
   static ThreadLocal::TypedSlot<Bootstrap::InternalListener::ThreadLocalRegistryImpl>*
       registry_tls_slot_;
+  static uint32_t buffer_size_;
```

Comment:
I think it would be good to add an explanation why this is static, e.g. client factory establishes the buffer while internal listener registry accepts the client connection. So the ownership is not followed - client factory needs to access config for listener registry.

Reply:
done